src/mui/delete/Delete.js

```diff
@@ -135,11 +136,11 @@ Delete.propTypes = {
 function mapStateToProps(state, props) {
     return {
         id: decodeURIComponent(props.match.params.id),
-        data:
-            state.admin.resources[props.resource].data[
-                decodeURIComponent(props.match.params.id)
-            ],
-        isLoading: state.admin.loading > 0,
+        data: getResourceRecord(
```

Comment:
why don't you always keep the same signature for selectors: `(state, props)`? It would avoid the higher order functions you use to always have the final `(state)` signature, and make memoization with `reselect` easier.